pkg/test/pods/pods.go

```diff
@@ -0,0 +1,78 @@
+package pods
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Pallinder/go-randomdata"
+	"github.com/imdario/mergo"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Options customizes a Pod.
+type Options struct {
+	Name             string
+	Namespace        string
+	Image            string
+	NodeName         string
+	ResourceRequests v1.ResourceList
+	NodeSelector     map[string]string
+	Tolerations      []v1.Toleration
+	Conditions       []v1.PodCondition
+}
+
+func defaults(options Options) *v1.Pod {
+	if options.Name == "" {
+		options.Name = strings.ToLower(randomdata.SillyName())
+	}
+	if options.Namespace == "" {
+		options.Namespace = "default"
+	}
+	if options.Image == "" {
+		options.Image = "k8s.gcr.io/pause"
+	}
+	if len(options.Conditions) < 1 {
```

Comment:
nit: == 0?

Reply:
Why is that better?

Reply:
it's more clear to me to see `== 0` rather than < 1 since it can't be negative.  Feel free to ignore, doesn't matter too much. I just stared at it for a few more seconds to try to understand why it's < 1 (if there were more cases than == 0)